install/kubernetes/helm/istio-init/files/crd-10.yaml

```diff
@@ -184,16 +184,15 @@ metadata:
   annotations:
     "helm.sh/resource-policy": keep
 spec:
-  group: rbac.istio.io
+  group: rbac.maistra.io
   names:
-    kind: ClusterRbacConfig
-    plural: clusterrbacconfigs
-    singular: clusterrbacconfig
+    kind: ServiceMeshRbacConfig
+    plural: servicemeshrbacconfigs
+    singular: servicemeshrbacconfig
     categories:
     - istio-io
     - rbac-istio-io
-  scope: Cluster
```

Comment:
this will have to be `scope: Namespaced`

Reply:
Will add that

Reply:
Defaults to `Namespaced`, so not really necessary to add it, apart from making it explicit.

Reply:
+1 but I believe the rest declare their scope so I've added it to make the scope explicit/obvious.

Reply:
It has a default value, but it's also a required field (not sure how those two fit together - I guess if you set it to an illegal value the default is used). I saw an error trying to create a CRD without this field